include/nbl/ext/ToneMapper/CToneMapper.h

```diff
@@ -87,16 +85,16 @@ class CToneMapper : public core::IReferenceCounted, public core::InterfaceUnmova
 		}
 
 		//
-		static core::SRange<const video::IGPUDescriptorSetLayout::SBinding> getDefaultBindings(video::IVideoDriver* driver, bool usingLumaMeter=false);
+		static core::SRange<const video::IGPUDescriptorSetLayout::SBinding> getDefaultBindings(video::ILogicalDevice* device, bool usingLumaMeter=false);
 
 		//
-		static inline core::smart_refctd_ptr<video::IGPUPipelineLayout> getDefaultPipelineLayout(video::IVideoDriver* driver, bool usingLumaMeter=false)
+		static inline core::smart_refctd_ptr<video::IGPUPipelineLayout> getDefaultPipelineLayout(video::ILogicalDevice* device, bool usingLumaMeter=false)
 		{
 			auto pcRange = getDefaultPushConstantRanges(usingLumaMeter);
-			auto bindings = getDefaultBindings(driver,usingLumaMeter);
-			return driver->createGPUPipelineLayout(
+			auto bindings = getDefaultBindings(device,usingLumaMeter);
+			return device->createGPUPipelineLayout(
 				pcRange.begin(),pcRange.end(),
-				driver->createGPUDescriptorSetLayout(bindings.begin(),bindings.end()),nullptr,nullptr,nullptr
+				device->createGPUDescriptorSetLayout(bindings.begin(),bindings.end()),nullptr,nullptr,nullptr
 			);
```

Comment:
shouldn't be static, should create in constructor and then cache.

Reply:
@nipunG314 get rid of all Descriptor Set utils from here, this is why HLSL accessor pattern is aawesome